routes/web.php

```diff
@@ -15,25 +15,19 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
+//     return view('welcome');
+// });
 
-Route::get('/dashboard', function () {
+Route::get('/', function () {
```

Comment:
Made base view dashboard view instead of having the link.  (/dashboard)

Reply:
Make sense.